src/extension.ts

```diff
@@ -52,6 +52,11 @@ export function activate(context: vscode.ExtensionContext) {
     ),
     vscode.commands.registerCommand(`${extensionName}.run-test`, extensionInstance.runTest),
     vscode.languages.registerCodeLensProvider(languages, extensionInstance.debugCodeLensProvider),
+    vscode.debug.registerDebugConfigurationProvider('node', extensionInstance.nodeDebugConfigurationProvider),
+    vscode.debug.registerDebugConfigurationProvider(
+      'vscode-jest-tests',
+      extensionInstance.snippetDebugConfigurationProvider
```

Comment:
you could just register the `extensionInstance.nodeDebugConfigurationProvider` here instead